applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -109,6 +112,17 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.nrf21540:
             device_flag += " -DSHIELD=nrf21540ek_fwd"
 
+        if options.custom_bt_name is not None and options.user_bt_name:
+            raise Exception("User BT name option is invalid when custom BT name is set")
+
+        if options.custom_bt_name is not None:
+            custom_bt_name = "_".join(options.custom_bt_name)[:MAX_USER_NAME_LEN].upper()
+            device_flag += " -DCONFIG_BT_DEVICE_NAME=\\\"" + custom_bt_name + "\\\""
+
+        if options.user_bt_name:
+            user_specific_bt_name = ("AUDIO_DEV_" + getpass.getuser())[:MAX_USER_NAME_LEN].upper()
+            device_flag += " -DCONFIG_BT_DEVICE_NAME=\\\"" + user_specific_bt_name + "\\\""
+
```

Comment:
```suggestion
        if options.custom_bt_name is not None 
            if options.user_bt_name:
                raise Exception("User BT name option is invalid when custom BT name is set")
            
            custom_bt_name = "_".join(options.custom_bt_name)[:MAX_USER_NAME_LEN].upper()
            device_flag += " -DCONFIG_BT_DEVICE_NAME=\\\"" + custom_bt_name + "\\\""
        else:
            if options.user_bt_name:
                user_specific_bt_name = ("AUDIO_DEV_" + getpass.getuser())[:MAX_USER_NAME_LEN].upper()
                device_flag += " -DCONFIG_BT_DEVICE_NAME=\\\"" + user_specific_bt_name + "\\\""

```

Reply:
Just to make it clearer that custom_name and user_name can't be set at the same time

Reply:
Now you are making me choose between you and Andreas. I think it should be pretty obvious that you can't both set your own name and have the program generate one for you. I will just leave it as is because it is more aesthetic i.m.o.